test_themes/__manifest__.py

```diff
@@ -48,7 +48,7 @@
         'web.assets_backend': [
             'test_themes/static/src/js/systray_items/website_switcher.js',
         ],
-        'web.assets_qweb': [
+        'website.assets_editor': [
```

Comment:
Any reason not to have both the xml and js file in the same bundle, like
```py
'web.assets_editor': [
    'test_themes/static/src/js/systray_items/*',
],
```
And maybe you can also remove the `js` folder to fit the community structure ?

Reply:
@rdeodoo Change done.

Reply:
Thank you, it was a bit nitpicking but at least now it is aligned with the rest :)